pkg/gameserverallocations/cache.go

```diff
@@ -21,6 +21,8 @@ import (
 )
 
 // gameserver cache to keep the Ready state gameserver.
+//
+//nolint:govet // ignore fieldalignment, low count
```

Comment:
I'm curious about some of these comments (especially the low count ones in struct names like this that _sound_ like we would have lots of them). 

/cc @markmandel

Reply:
This one in particular is embedded in AllocationCache. I changed the comment and the name of the structure to reflect that, since it was also confusing me.

The gameServerStateCache was different and I was wrong there after looking at it more - I thought it followed the pattern here. I went ahead and just fixed the alignment.